Lesson20/Mesut/app.js

```diff
@@ -1,35 +1,32 @@
-// Get the input field, form and the list
-const input = document.getElementById('todo-input');
-const list = document.getElementById('todo-list');
-const form = document.getElementById('todo-form');
+const input = document.getElementById("todo-input");
+const list = document.getElementById("todo-list");
+const form = document.getElementById("todo-form");
 
-// Add a listener to the form to handle form submission
-form.addEventListener('submit', (event) => {
+form.addEventListener("submit", (event) => {
   event.preventDefault();
-  // Create a new task item and add it to the list
 
-  const listItemElement=document.createElement("li");
-  listItemElement.innerText = input.value;
-  
-  // Remove button
-  const removeButton = document.createElement("button");
-  removeButton.innerText = 'X';
-  // TODO: Update styles of removeButton! (use button--light class)
-  listItemElement.appendChild(removeButton);
+  if (input.value !== "") {
+    const divElement = document.createElement("div");
+    divElement.classList.add("container-div");
+    const listItemElement = document.createElement("p");
+    listItemElement.className = "list-item";
```

Comment:
It's ok to use `listItemElement.className` in this case but be aware that if you had other classes set for the elements, this code would override any existing classes. Therefore, it's more common to use `element.classList.add();` in real life to avoid unexpected behaviour.